js/script.spec.js

```diff
@@ -24,5 +24,15 @@ describe("Beer display module", () => {
     test("createAbvParameter with selected option 10 returns false", () => {
         expect(createAbvParameter(10)).toBeFalsy();
     });
+
+    test("createBeerContainer is in a div tag", () => {
+        const result = createBeerContainer().nodeName;
+        expect(result).toBe('DIV');
```

Comment:
Consider merging these two test in one test

Reply:
I would like to keep test cases to be as simple as possible, single responsible. It does not affect performance much, meanwhile it looks cleaner to me.